### ISSUE TITLE
Meteor middlewares as npm package

### ISSUE THREAD

**Issue by michaltakac** [djhi/my-nutrition#17]
Did you consider packaging those middlewares actionTypeBuilder into NPM package?

**Comment by djhi** [comment]
Yes, I did. Actually, we use this pattern at work (see [marmelab/javascript-boilerplate](https://github.com/marmelab/javascript-boilerplate)). I'll make a package out of it soon.

**Comment by michaltakac** [comment]
Great, I'm building Meteor boilerplate for my next projects and it's heavily inspired by `my-nutrition` app and this gist: https://gist.github.com/dbismut/35b89185998d68bc18fd

P.S.: Bookmarking `javascript-boilerplate`, looks good.
Thank you.

**Comment by Naycon** [comment]
+1 for making an NPM package out of the meteor middleware. This was exactly what I was looking for. 

The whole repo is fantastic for learning how to use Meteor with Redux, thanks for publishing it.

**Comment by michaltakac** [comment]
Would be interesting to see this repo refractored for new Meteor 1.3 folder structure, I think people using Meteor + React + Redux would benefit significantly.

**Comment by djhi** [comment]
@michaltakac agreed, unfortunatly, I just don't have time currently.

**Comment by samylaumonier** [comment]
I've been waiting a while for a package, but I finally decided to create it myself because you don't have enough time to do it.

[You can find it here.](https://github.com/samybob1/meteor-redux-middlewares)

Regards,
Samy

**Comment by djhi** [comment]
Good job ! I don't work with meteor anymore so I it would have been hard to find time for this. Thanks @samybob1 !
